src/components/Landing.js

```diff
@@ -76,9 +76,51 @@ export default class Landing extends Component {
       this.setError(`[${item.type}] ${item.error}`);
     });
   }
+  processSummaryErrors(summary) {
+    if (!summary) return false;
+    //PDMP medications
+    let pdmpMeds = summary["PDMPMedications"];
+    if (pdmpMeds && pdmpMeds["PDMPMedications"]) {
+      let o = pdmpMeds["PDMPMedications"];
+      o.forEach(item => {
+        //look for medication that contains NDC code but not RxNorm Code
+        if (item["NDC_Code"] && !item["RXNorm_Code"]) {
+          this.setError(`Medication, ${item["Name"]}, did not have an MME value returned, total MME and the MME overview graph are not reflective of total MME for this patient.`)
+        }
+
+      })
+    }
+  }
   setError(message) {
     if (!message) return;
     this.errorCollection.push(message);
+    this.writeErrorToLog(message);
+  }
+  writeErrorToLog(message) {
+    if (!message) return;
+    const auditURL = `${getEnv("REACT_APP_EPIC_SUPPORTED_QUERIES")}/auditlog`;
```

Comment:
The base URL of the confidential backend should be configured as `REACT_APP_CONF_API_URL`

```suggestion
    const auditURL = `${getEnv("REACT_APP_CONF_API_URL")}/auditlog`;
```